style.css

```diff
@@ -0,0 +1,576 @@
+/************ CSS Reset **********/
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+/******************************/
+
+/*****************************************/
+body {
+  font-family: "Times New Roman", Times, serif;
+  font-size: 16px;
+  font-stretch: 100%;
+  font-weight: 400;
+  line-height: 16px;
+  max-width: 1440px;
+  overflow-x: hidden;
+}
+a{
+    color: rgb(98, 136, 165);
+}
+header {
+  width: 100%;
+  height: 42px;
+  background-color: bisque;
+}
+button {
+  font-family: Arial, Helvetica, sans-serif;
+  background: none;
+  border: none;
+  box-sizing: border-box;
+}
+nav {
```

Comment:
#### `HINT`

Add `z-index` property with an `<integer>` value to the `nav` element. This makes it render on top of elements with `position: absolute`. See current display below:
![nav index render](https://user-images.githubusercontent.com/36057474/63581248-87730200-c58e-11e9-859c-040a6986e83a.gif)

Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index).